src/components/molecules/Menu/Menu.tsx

```diff
@@ -15,6 +17,7 @@ const useStyles = makeStyles({
 const Menu = () => {
   const classes = useStyles();
   const handleLogout = useLogout();
+  const { channels, DMs } = useAppState();
 
   const Choices = ['Game', 'DM', 'Channel', 'Community', 'Profile'];
   const RenderChoices = Choices.map((item) => (
```

Comment:
이 PR에서 수정된 것은 아니지만, 코드의 통일성을 위해서 이렇게 하는 것이 나을까요?
```suggestion
  const RenderChoices = Choices.map((item) => (
  <Grid item key={item}>
      <NavLink
        activeStyle={{ color: 'yellow' }}
        style={{ textDecoration: 'none', color: 'inherit' }}
        to={`/${item.toLowerCase()}`}
      >
```